opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -237,6 +246,11 @@ def _set_start_end_times(self, point, record, metric_descriptor):
                     self._last_updated[updated_key] + int(1e6),
                     NANOS_PER_SECOND,
                 )
+        else:
+            (
+                point.interval.start_time.seconds,
+                point.interval.start_time.nanos,
+            ) = (seconds, nanos)
```

Comment:
do we even need to set start_time if not cumulative?

Reply:
No, this value doesn't need to be set since Monitoring will take end_time==start_time. This is here just for clarity.